frontend/justfile

```diff
@@ -40,7 +40,12 @@ wait-up: up
 # Set up user and test site in Plausible
 init: wait-up
     cd .. && ./setup_plausible.sh
-
+    
+recreate:
+    just ../down -v
+    just up "--force-recreate --build"
+    just init
```

Comment:
These steps are for the Docker-based services. The frontend is not using Docker so the steps will be different. See the 'Description' section of issue #2028 for the steps to perform.

Reply:
Added the required changes, kindly review it.